packages/app/src/runner/SpecRunnerHeaderOpenMode.vue

```diff
@@ -23,14 +23,16 @@
         >
           <i-cy-crosshairs_x16 :class="[selectorPlaygroundStore.show ? 'icon-dark-indigo-500' : 'icon-dark-gray-500']" />
         </Button>
-        <a
+        <input
```

Comment:
Here is the `<input>` I was talking about.

Reply:
Why did we need to change this to an input? I think I missed the reason

Reply:
This is here: https://github.com/cypress-io/cypress/pull/23413#discussion_r949976076

I think it was an input in Cypress 9, which I always didn't really love - I'm hoping when we clean up the UX for this, this will only render an input when it is actually editable in studio mode, and will render a plain old link the rest of the time.

Reply:
I guess `<input>` as a link is not really the best a11y. We could do a fancy switcharoo depending if studio is active? Or, hack `<a>` to be editable (but this seems not ideal).

I had a ton of problems with tailwind doing weird things - for some reason the sizing changes between `<a>` and `<input>` and messed up the flow of the flex-wrap, even though tailwind claims to do a full CSS reset. Probably need to jam with a CSS-minded individual to sort this out.